docs/contributing/stacks.md

```diff
@@ -62,7 +63,7 @@ git init
 git add .
 git commit -m 'Seed repo'
 git remote add origin <url from github>
-git push -u origin master
+git push -u origin main
```

Comment:
We haven't yet renamed to use `main` in this project. I suggest we don't go for such change in this PR already doing other changes and that the master->main transition can be quite significant by itself.

Reply:
This is different. This part of Documentation describes how to create your own github repo and your image.
By default, new GitHub repos will have `main` branch, that's why I changed this.

Reply:
Ahhj!